xsuite/package.json

```diff
@@ -53,6 +53,9 @@
     "typescript": "5.4.0-beta",
     "vitest": "^1.2.2"
   },
+  "optionalDependencies": {
```

Comment:
My intent was to not install this on `npm install` since the package is quite large.
`npm install --omit=optional` can be used to install and omit these however.

I would still keep this here since it is not mandatory for the xsuite package to function properly and I would rather have a user of this library manually do `npm install @xsuite/chainsimulator` if he needs to use this package. (even though that is not enforced)

Reply:
People are going to run `npm install`, right? So they will install the dependency anyway.

Reply:
Moved to `dependencies`